skipruntime-ts/examples/sum.ts

```diff
@@ -4,6 +4,19 @@ import { ManyToOneMapper } from "@skipruntime/core";
 
 import { runService } from "@skipruntime/server";
 
+let platform: "wasm" | "native" | undefined;
+
+switch (process.env["SKIP_PLATFORM"]) {
+  case "wasm":
+    platform = "wasm";
+    break;
+  case "native":
+    platform = "native";
+    break;
+  default:
+    break;
+}
```

Comment:
Why let the platform be undefined?  This is semantically equivalent to the simpler:

```suggestion
const platform : "wasm" | "native" = process.env["SKIP_PLATFORM"] == "native" ? "native" : "wasm";
```

Reply:
Just to make typescript happy, `Type 'string' is not assignable to type '"wasm" | "native" | undefined'`